src/openforms/submissions/tests/test_submission_report.py

```diff
@@ -97,6 +99,35 @@ def test_report_generation(self, mock_request):
         # report.content.name contains the path too
         self.assertTrue(report.content.name.endswith("Test_Form.pdf"))
 
+    def test_report_is_generated_in_same_language_as_submission(self):
+        submission = SubmissionFactory.create(
+            language_code="en",
+            completed=True,
+            form__name_en="Translated form name",
+        )
+        FormStepFactory.create(
+            form=submission.form,
+            form_definition__name="Walsje",
+            form_definition__name_en="A Quickstep",
+            form_definition__configuration={
+                "components": [
+                    {
+                        "key": "foo",
+                        "type": "textfield",
+                        "label": "Foo",
+                        "showInEmail": True,
+                    }
+                ],
+            },
+        )
```

Comment:
I'd consider using the chained `form__generate_minimal_setup` in the `SubmissionFactory` to stuff everything into a single factory call. CTRL+F on the codebase to find some examples.

Additionally, if a submission `language_code` is provided different from `settings.LANGUAGE_CODE`, this should also enforce `form__translations_enabled=True` to generate consistent data. That should be possible with something like:

```py
class SubmissionFactory(...):
    form = factory.SubFactory(
        FormFactory,
        translation_enabled=factory.LazyAttribute(lambda o: o.factory_parent.language_code != settings.LANGUAGE_CODE)
    )
```

Reply:
I expected `submission__form__form_step__form_definition__configuration` to work, but had to change it to `submission__form__formstep__form_definition__configuration`... Where is this inconsistent black wizardry done?

Reply:
`form.formstep_set` is a reverse lookup, and the `generate_minimal_setup` is a Trait that sets up the `formstep` kwarg which generates essentially the m2m through table entry (and exposes the underlying data via that name): https://github.com/open-formulieren/open-forms/blob/master/src/openforms/forms/tests/factories.py#L26